### ISSUE TITLE
More than one cover?

### ISSUE THREAD

**Issue by MPconta** [abarisain/dmix#471]
For all my stored CD's i have not only the front-cover, but also the back-cover and a inside-cover stored in the library.
Is it possible to see them at the now-listening-screen?
For example: When the screen is thouched, the picture changes to the next in the list.

Ok, sorry about that wish - i am sure, it would not be easy to implement a function like that. :-)

But at least - could you repair the actual cover-view in the now-listening-screen? When the picture changes its size because of rotation of the tablet, it gets very fuzzy!

**Comment by wonea** [comment]
Yeah, massively. Being able to browse covers, then dive right in would really make a tablet experience.
